SchedulePlatform.Service/Models/Customer/CustomerCreateModel.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.ComponentModel.DataAnnotations;
+
+namespace SchedulePlatform.Service.Models.Customer
+{
+    public class CustomerCreateModel
+    {
+        public Guid Id = Guid.NewGuid();
+        [StringLength(50, MinimumLength = 3, ErrorMessage = "First Name should be minimum 3 characters and a maximum of 50 characters")]
```

Comment:
Add some new lines between these properties and their validations, please. It will be easer to read. 
```
        [DataType(DataType.EmailAddress)]
        public string? Email { get; set; }
        [DataType(DataType.PhoneNumber)]
        public string? Phone { get; set; }
        [Range(1, 2.4, ErrorMessage = "Height should be between 1.0m  and 2.4 m ")]
        public decimal Height { get; set; }
        [Range(1,250,ErrorMessage ="Weight should be higher than 1 kg ")]
        public decimal Weight { get; set; }
```
vs.
```
        [DataType(DataType.EmailAddress)]
        public string? Email { get; set; }

        [DataType(DataType.PhoneNumber)]
        public string? Phone { get; set; }

        [Range(1, 2.4, ErrorMessage = "Height should be between 1.0m  and 2.4 m ")]
        public decimal Height { get; set; }

        [Range(1,250,ErrorMessage ="Weight should be higher than 1 kg ")]
        public decimal Weight { get; set; }
```